src/app/clusters/test-cluster-server/test-cluster-server.cpp

```diff
@@ -137,6 +145,36 @@ SimpleEnum gSimpleEnums[kAttributeListLength];
 size_t gSimpleEnumCount = 0;
 Structs::NullablesAndOptionalsStruct::Type gNullablesAndOptionalsStruct;
 
+void emberAfUnitTestingClusterTestBatchWork(AsyncBatchCommandsWorkData * asyncWorkData)
```

Comment:
Please do not add `emberAf` to new code if possible, since this is local to this cluster.